### PR TITLE
edits the readme

### PR DESCRIPTION
**Pull request by wgahnagl**
- adds badges, images and links to the readme 
- adds links to the docs and IRC
- adds writeup from the main website

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,100 @@
...
+
+If you don't have a device running AsteroidOS, instructions for installation can be found [here](https://asteroidos.org/install/) 
+
+### Development Instructions 
+To get started with a developing for AsteroidOSSync, fork and clone the project, and import into any Android IDE. 
+
+# Roadmap 
+See the [open issues](https://github.com/AsteroidOS/AsteroidOSSync/issues) for a list of proposed features and known issues 
+
+# Contibruting
```
Comment (jrtberlin) [inline README.md]: "Contibruting": Can you please fix this typo and clean up the commit history.

**Comment by jrtberlin** [comment]
Hi, thank you for contributing to the project. Your changes are improving the readme a lot and I would love to merge them! It would be great if you could fix the typo before I do. Thanks!

File: README.md (hunk 1 above)
Comment (wgahnagl) [reply README.md]: ah! fixed!

**Comment by jrtberlin** [comment]
Thank you for your contribution. :tada:

**Pull request merged**